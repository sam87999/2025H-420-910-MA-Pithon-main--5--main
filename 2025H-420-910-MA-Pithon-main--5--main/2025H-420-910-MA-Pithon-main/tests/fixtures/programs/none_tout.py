# Test de Alexandrine Dub√©
print(None in [1, 2, 3])
print(None or True)
print(None and False)
print(not None)
print(None == 4)
print(None != 4)
print(None)
print(None or None)
print(None and None)
print(None == None)
print(None != None)
