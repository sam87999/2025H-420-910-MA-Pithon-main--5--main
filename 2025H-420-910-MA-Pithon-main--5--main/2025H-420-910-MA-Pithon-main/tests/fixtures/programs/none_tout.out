False
True
None
True
False
True
None
None
None
True
False
